db/tweets.sql

```diff
@@ -0,0 +1,42 @@
+-- USERDATA table for holding the user information (USERNAME, user full name, and user ID)
+CREATE TABLE USERDATA(
+    USER_ID NUMBER NOT NULL,
+    USERNAME VARCHAR2(255 char) NOT NULL,
+    USERFULLNAME VARCHAR2(255 char),
+    PRIMARY KEY (USER_ID)
+);
+
+COMMENT ON TABLE USERDATA IS 'Contains the data for the user, e.g (USER ID, USERNAME, USER FULL NAME)';
+COMMENT ON COLUMN USERDATA.USER_ID IS 'ID OF users';
+COMMENT ON COLUMN USERDATA.USERNAME IS 'Username name';
+COMMENT ON COLUMN USERDATA.USERFULLNAME IS 'User full name';
+
+-- sequence counter for the unique user ID, starts from 1 and increment by 1
+CREATE SEQUENCE seq_USER_ID
+MINVALUE 1
+START WITH 1
+INCREMENT BY 1
+CACHE 10;
+
+--INSERT INTO USERDATA (USER_ID)
+--VALUES (seq_USER_ID.nextval);
```

Comment:
Can you try adding some example data to both tables? For example: https://docs.oracle.com/cd/B28359_01/server.111/b28318/datatype.htm#CNCPT413

You will have to implement GET endpoint first so we will need some data in the database.